app/schema.py

```diff
@@ -12,13 +12,38 @@ class Meta:
         model = User
 
 
+class FriendRequestType(DjangoObjectType):
+    class Meta:
+        model = FriendRequest
+
+
+class FriendType(DjangoObjectType):
+    class Meta:
+        model = Friend
+
+
 class Query(ObjectType):
     user = graphene.Field(UserType, contact=graphene.String())
+    friend_requests_sent = graphene.List(FriendRequestType)
+    friend_requests_received = graphene.List(FriendRequestType)
+    friends = graphene.List(FriendType)
 
     @login_required
     def resolve_user(self, info, **kwargs):
         return info.context.user
 
+    @login_required
+    def resolve_friend_requests_sent(parent, info, **kwargs):
+        return [ fr_req for fr_req in FriendRequest.objects.filter(from_user=info.context.user) ]
+    
+    @login_required
+    def resolve_friend_requests_received(parent, info, **kwargs):
+        return [ fr_req for fr_req in FriendRequest.objects.filter(to_user=info.context.user) ]
+
+    @login_required
+    def resolve_friends(parent, info, **kwargs):
+        return [ fr for fr in Friend.objects.filter(current_user=info.context.user) ]
```

Comment:
`FriendRequest.objects.filter(current_user=info.context.user)` is enough since it returns an iterable anyway